ogr/services/github/comments.py

```diff
@@ -29,28 +29,34 @@
 
 
 class GithubIssueComment(IssueComment):
-    def __init__(self, raw_comment: _GithubIssueComment) -> None:
-        self.__raw_comment = raw_comment
-        super().__init__(
-            comment=raw_comment.body,
-            author=raw_comment.user.login,
-            created=raw_comment.created_at,
-            edited=raw_comment.updated_at,
-        )
+    def __init__(self, raw_comment: _GithubIssueComment = None, **kwargs) -> None:
+        if raw_comment is not None:
+            self.__raw_comment = raw_comment
+            super().__init__(
+                comment=raw_comment.body,
+                author=raw_comment.user.login,
+                created=raw_comment.created_at,
+                edited=raw_comment.updated_at,
+            )
+        else:
+            super().__init__(**kwargs)
```

Comment:
Hmm, interesting. (Compatible with everything..;-)

Personally, I don't like that we don't see the real attributes in the `__init__`. (Only the `**kwargs**`.)

Is there a situation, where we want to use it? I am just worried about the situation, where we have two kinds of this object that needs to be handled differently.

Reply:
Can you at least explicitly name the arguments? I know it's not necessary, but for the documentation purposes.

You know. *"Explicit is better than implicit."*